pkg/sentry/config/config.go

```diff
@@ -0,0 +1,141 @@
+package config
+
+import (
+	"encoding/json"
+	"errors"
+	"fmt"
+	"os"
+	"time"
+
+	scheme "github.com/dapr/dapr/pkg/client/clientset/versioned"
+	dapr_config "github.com/dapr/dapr/pkg/config"
+	"github.com/dapr/dapr/utils"
+	log "github.com/sirupsen/logrus"
+	meta_v1 "k8s.io/apimachinery/pkg/apis/meta/v1"
+)
+
+const (
+	RootCertFilename            = "trust.pem"
+	IssuerCertFilename          = "crt.pem"
+	IssuerKeyFilename           = "key.pem"
+	kubernetesServiceHostEnvVar = "KUBERNETES_SERVICE_HOST"
+	kubernetesConfig            = "kubernetes"
+	selfHostedConfig            = "selfhosted"
```

Comment:
should this be consistent with standalone mode?

Reply:
See above comment